adapter/memQuota/memQuota.go

```diff
@@ -73,7 +73,11 @@ func (builder) ValidateConfig(cfg adapter.Config) (ce *adapter.ConfigErrors) {
 	return
 }
 
-func (builder) NewQuotasAspect(env adapter.Env, c adapter.Config, d map[string]*adapter.QuotaDefinition) (adapter.QuotasAspect, error) {
+func (builder) Configure(quotas map[string]*adapter.QuotaDefinition) error {
```

Comment:
And presumably QuotaDefinition will be replaced with some other type (I forgot the suffix we chose)